src/nvim/search.c

```diff
@@ -2411,6 +2412,18 @@ static int cls(void)
 {
   int c;
 
+  // Treat concealed characters like whitespace.
+  if (curwin->w_p_cole > 0 && curwin->w_cursor.col > 0
+      && curbuf->b_p_smc > 0 && curwin->w_cursor.col < curbuf->b_p_smc
+      && conceal_cursor_line(curwin)) {
+    syn_get_id(curwin, curwin->w_cursor.lnum, curwin->w_cursor.col,
+               false, NULL, false);
+    int syntax_flags = get_syntax_info(&c);
+    if (syntax_flags & HL_CONCEAL) {
+      return 0;
+    }
```

Comment:
`&c` is not used. If you just need a dummy var I think we would normally create a separate dummy var.